### DISCUSSION TITLE
Berkas Digital

### DISCUSSION THREAD

**Discussion by muhariefnur** [basoro/Khanza-Lite#152] [Q&A]
Assalamu'alaikum
Bagaimana cara hapus berkas digital ?

Terimakasih

**Comment by basoro** [comment]
Untuk sementara bisa dilihat di Modul Vedika. Disana disediakan tombol Hapus Berkas Digital. Karena kewenangan hapus data Rekam Medik ada di user dengan Role rekammedis.

Salam
